### ISSUE TITLE
Difference between this implementation and PyTorch team's version?

### ISSUE THREAD

**Issue by chrischen** [iamRusty/fast-neural-style-pytorch#22]
What is the difference of this implementation (regarding just static image style transfer, not for videos) vs the PyTorch teams implementation: the PyTorch Team[PyTorch Examples: fast-neural-style](https://github.com/pytorch/examples/tree/master/fast_neural_style)?

**Comment by rrmina** [comment]
They are basically the same except for 2 major things:

1. This repo uses the original VGG networks used in the [original fast-neural-style paper](https://arxiv.org/abs/1603.08155). In contrast, the Pytorch Team example uses VGG networks from Pytorch Image Model Zoo. 
2. This repo uses (1) Content Loss, (2) Style Loss, and (3) Total Variation Loss, **while** Pytorch Team example uses only (1) and (2). Total Variation Loss smooths the pixels of the output image by minimizing the difference of pixel values of adjacent pixels.

TL;DR This repo is a more faithful reproduction of the [original fast-neural-style paper](https://arxiv.org/abs/1603.08155), but both this and Pytorch's should more or less have similar results! :)))
